internal/firefox.py

```diff
@@ -1044,7 +1044,9 @@ def calculate_page_stats(self, requests):
             elif request['responseCode'] > -1:
                 page['responses_other'] += 1
             if main_request is None and \
-                    (request['responseCode'] == 200 or request['responseCode'] == 304):
+                    (request['responseCode'] == 200 or request['responseCode'] == 304) and \
+                     request['contentType'] != 'application/ocsp-response' and \
+                     request['contentType'] != 'application/pkix-crl':
```

Comment:
Can you also add a check to make sure "'contentType' in request" before checking the values?  Otherwise it will throw an exception for responses without a contentType.  It doesn't look like the code above has a default value for it when it's not present so it just won't be populated.

Reply:
You are right, I forgot about that. I added a check if the key doesn't exist. The version before my PR accepted a request without contentType as an main request, so I assume that this is the desired behavior. I changed my code to match the old behavior. Now a request without contentType or a request with the right contentType can be the main request.